src/components/UserDropdown/index.tsx

```
@@ -115,16 +122,16 @@ function UserDropDown({ user, onClickLogout }: UserDropDownProps) {
                 <div className="p-2">
                     {filteredMenuItems.map((item, index) => {
                         const Icon = item.icon
-                        const isStaffOnly = item.roles && item.roles[0]?.includes("Staff") && !item.roles[0]?.includes("Guest")
+                        const isStaffOnly = item.roles && item.roles?.[0]?.includes("Staff") && !item.roles?.[0]?.includes("Guest")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)